modules/serverless-logic-running-workflows.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * serverless/serverless-logic/serverless-logic-creating-managing-workflows.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="serverless-logic-running-workflows_{context}"]
+= Running a workflow locally
+
+You can use the `run` command with `kn workflow` to build and run your {ServerlessLogicProductName} workflow project in your current directory.
+
+
+.Prerequisites
+
+* You have installed Podman on your local machine.
+* You have installed the {ServerlessLogicProductName} `kn-workflow` CLI plugin.
+* You have created an {ServerlessLogicProductName} workflow project.
+
+.Procedure
+
+. Run the following command to build and run your {ServerlessLogicProductName} workflow project:
++
+[source,terminal]
+----
+$ kn workflow run
+----
++
+When the project is ready, the Development UI automatically opens in your browser on `localhost:8080/q/dev-ui`. 
```

Comment:
```suggestion
When the project is ready, the Development UI automatically opens in your browser on `localhost:8080/q/dev-ui`. In that screen you must find the `Serverless Workflow Tools` card.
Alternatively, you can use this link: http://localhost:8080/q/dev-ui/org.apache.kie.sonataflow.sonataflow-quarkus-devui/workflows
```

Reply:
For you to understand what we try to say.

The Development UI opens automatically, and user needs to find the "Card" below to get access to the workflows.
![Screenshot from 2024-07-12 13-04-27](https://github.com/user-attachments/assets/ec18c942-8bdc-4947-9249-9b639e2595b1)


Alternativelly, he can go the workflows directly using 
localhost:8080/q/dev-ui/org.apache.kie.sonataflow.sonataflow-quarkus-devui/workflows